Array/Array Rotation/right_side_rotation.py

```diff
@@ -0,0 +1,16 @@
+#right rotation of array
+#it avoids unnecessary no. of recursions for large no. of rotations.
+a=[1,2,3,4,5]
+def right_rot(arr,r): # r is the no. of times to rotate
+  if r<len(arr):
+    s=r
+  else:
+    s=r%len(arr)
```

Comment:
if you use function like len(arr) multiple time, store it in a locl/global variable, this will hugely reduce time complexity.